src/cascadia/TerminalControl/ControlInteractivity.cpp

```diff
@@ -44,9 +46,16 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         _lastMouseClickPos{},
         _selectionNeedsToBeCopied{ false }
     {
+        _id = ControlInteractivity::_nextId++;

```

Comment:
nit: I think the existence of the operators on atomic structs is a bit of a mistake because they hide both the underlying behavior (fetch-load vs. load-fetch) and the memory model (relaxed, acquire, release, sequentially consistent). I would suggest using this:
```cpp
_id = _nextId.fetch_add(1, std::memory_order_relaxed);
```